index.qmd

```diff
@@ -28,3 +28,7 @@ the
 [Azure DevOps for Analysis guide](https://dfe-analytical-services.github.io/azure-devops-for-analysis/).
 ::::
 
+There is a 
```

Comment:
Just a slight change to the text that I think just makes it work a little better:
```suggestion
If at any point in this tutorial you're unsure of any terminology, there is a 
```